### ISSUE TITLE
[BUG] rancher-monitoring-grafana is too small and it keeps growing

### ISSUE THREAD

**Issue by abonillabeeche** [harvester/harvester#2282]
**Describe the bug**
The rancher-monitoring-grafana volume keeps growing but it's size should remain the same. Since the volume is low in size, it eventually grows more than it should.

**To Reproduce**
1. Turn Off Harvester servers.
2. Turn On Harvester servers.
3. Eventually the volume grows over the assigned 10Mi

**Expected behavior**
The volume should be created with a size that doesn't cause trouble should be encountered and the volume should not be growing so drastically.

**Environment:**
 - Harvester ISO version: 1.0.1 and 1.0.2
 - Underlying Infrastructure: R620

**Additional context**
The only action I do that is unusual, is that I power off/on the servers frequently to preserve power.

**Comment by w13915984028** [comment]
Under investigation.
```
Option A:
 keep pvc, 
 set default size to 10Gb according to grafana's recommended default value,
 expand pvc when upgrading
 
option B:
  remove PVC for grafana, by default, it is not needed

```

Option B also patially improve #2013

**Comment by TachunLin** [comment]
Verified **fixed** on `master-3bcba9d3-head` (09/13). Close this issue.

### Result
* The `rancher-monitoring-grafana` default allocated with `2Gi` and Actual usage `108 Mi` after running after 24 hours
  ![image](https://user-images.githubusercontent.com/29251855/191000121-9c3c640e-7d7f-4d1b-84f6-39745abca0ce.png)

* Turn off then turn on the specific vip harvester node machine, the The `rancher-monitoring-grafana` keep stable in `107 Mi` after turning on 60 minutes
  ![image](https://user-images.githubusercontent.com/29251855/191011632-06f03787-6bac-4bff-b964-787b414407ab.png)

* Turn off then turn on all four harvester node machines, the The `rancher-monitoring-grafana` keep stable in `107 Mi` after turning on 60 minutes
![image](https://user-images.githubusercontent.com/29251855/191023195-661275d3-1edb-4739-b585-cca23d17a3d9.png)


### Test Information
* Test Environment: 3 nodes harvester on provo bare machine
* Harvester version: `master-3bcba9d3-head` (09/13)

### Verify Steps
1. Harvester cluster running after 24 hours
2. Access Harvester Longhorn dashboard via https://<vip>/dashboard/c/local/longhorn
3. Open the Longhorn UI
4. Open the volume page
5. Check the `rancher-monitoring-grafana` size and usage 
6. Shutdown a management node machine
7. Power on the management node machine
8. Wait for 60 minutes
9. Check the `rancher-monitoring-grafana` size and usage in Longhorn UI
10. Shutdown all management node machines in sequence 
11. Power on all management node machines in sequence
12. Wait for 60 minutes
13. Check the `rancher-monitoring-grafana` size and usage in Longhorn UI
